product_security/red_hat_responsibilities/access_control.adoc

```diff
@@ -0,0 +1,105 @@
+[[security-access-control]]
+= Access Control
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+[[security-authentication]]
+== Authentication
+
+Authentication for the cluster is configured as part of the on-boarding process.
+{product-title} includes a built-in OAuth server. The OAuth server
+generates two kinds of tokens:
+
+* _Access tokens_ are longer-lived tokens that grant access to the API.
+* _Authorize codes_ are short-lived tokens; their only use is to be exchanged for
+an access token.
+
+You can integrate your cluster with various identity providers at installation
+time, or later by opening a support case.
+
+Supported identity providers include:
+
+* Keystone
+* Google Auth
+* GitHub OAuth
+* OpenID Connect
+* LDAP
+
+[[security-encryption-and-key-management]]
+== Encryption and Key Management
+
+By default, a new internal public key infrastructure (PKI) is created for each
+deployment of {product-title}. The internal PKI uses 2048-bit RSA keys and
+SHA-256 signatures. The platform PKI is dedicated to managing the platform
+components’ certificates. This includes: the master API, controllers, and nodes.
+Also, by default, most of the infrastructure components deployed at install time
+are signed by this CA; for example: metrics, logging, and the default router
+wildcard certificate. By default, the PKI CA related files are stored on all
+OpenShift masters in the /etc/origin/master directory. The etcd cluster has its
+own PKI.
+
+Encryption keys for persistent volumes are managed by the Key Management service
+of the underlying infrastructure solution.
+
+[[security-authorization]]
+== Authorization
+
+[[security-RBAC]]
+=== Role-based Access Control (RBAC)
+
+* xref:../../architecture/additional_concepts/authorization.adoc#architecture-additional-concepts-authorization[Learn
+how authorization is managed].
+
+* xref:../../admin_guide/manage_rbac.adoc#admin-guide-manage-rbac[Learn about
+Managing Role-based Access Control (RBAC)].
+
+[[security-cluster-roles-local-roles]]
+=== Cluster Roles and Local Roles
+
+Roles are collections of policy rules, which are sets of permitted verbs that
+can be performed on a set of resources. {product-title} includes a set of
+default cluster roles that can be bound to users and groups cluster wide or
+locally.
+
+[[security-scc]]
+=== Security Context Constraints
+
```

Comment:
Add text below SCC heading:
Security context constraints allow administrators to control permissions for pods. As an OpenShift Dedicated cluster administrator, you can list and view details for SCCs, but cannot edit or delete the default SCCs.